examples/gno.land/p/demo/gnofundme/gnofundme.gno

```diff
@@ -0,0 +1,124 @@
+package gnofundme
+
+import (
+	"std"
+	"strconv"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/avl"
+)
+
+type Campaign struct {
+	title        string
+	description  string
+	goal         uint
+	current      uint
+	begin        time.Time
+	deadline     time.Time
+	owner        *User
+	beneficiary  *User
+	contributors []*Contributor
+}
+
+func NewCampaign(title string, description string, goal uint, begin time.Time, deadline time.Time, owner *User, beneficiary *User) *Campaign {
```

Comment:
How will you check if the string variables aren't empty? I think it would be great if you add some conditionals to avoid this issue. Maybe a minimum of 5 chars for the title, 10 for description, a goal of more than 0, etc.

Reply:
Hey, I was wondering if I add my checks on the realm (can cost more gas) or in the front (react), is it secure to do it on the front?

Reply:
If you have a safe testing of your front, it should be ok :-)